index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require "functions.php";
+
+$db = getDB();
+$pokemons = getPokemons($db);
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pokedex</title>
+    <link rel="stylesheet" href="./css/main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Finger+Paint&family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet">
+    <link rel="icon" href="https://pngimg.com/uploads/pokeball/pokeball_PNG21.png">
+    <link rel="stylesheet" href="resources/css/normalize.css">
```

Comment:
Put normalize.css before your main.css. This way it won't override any styles you have created.